stl/inc/format

```diff
@@ -1428,10 +1538,18 @@ _NODISCARD _OutputIt _Write_aligned(_OutputIt _Out, const int _Width, const _Bas
         }
     }
 
-    // TRANSITION, add support for unicode/wide formats
-    _Out = _RANGES fill_n(_STD move(_Out), _Fill_left, _Specs._Fill[0]);
+    const basic_string_view<_CharT> _Fill_char{_Specs._Fill, _RANGES find(_Specs._Fill, '\0')};
+    for (; _Fill_left > 0; --_Fill_left) {
+        _Out = _RANGES copy(_Fill_char, _STD move(_Out)).out;

```

Comment:
Ditto `_RANGES _Unchecked_copy`

Reply:
Ditto "I see no benefit commensurate with the additional code". Do you?